docs/en/solutions/ArtifactPromotionRun_Approval_Notification_with_CorpWeChat.md

```
@@ -0,0 +1,493 @@
+| Event Type | CloudEvent Type | Trigger Scenario | Notification Recipients |
+|---------|----------------|----------|---------|
+| Approval Pending | `dev.katanomi.cloudevents.kubeevent.artifactpromotionrun.approvalpending.v1alpha1` | Artifact promotion request pending approval | Approvers |
+| Approval Denied | `dev.katanomi.cloudevents.kubeevent.artifactpromotionrun.approvaldenied.v1alpha1` | Promotion request has been rejected | Requester |
+| Approval Approved | `dev.katanomi.cloudevents.kubeevent.artifactpromotionrun.running.v1alpha1` | Promotion request approved and now executing | Requester |
+| Promotion Failed | `dev.katanomi.cloudevents.kubeevent.artifactpromotionrun.failed.v1alpha1` | Promotion operation has failed | Requester |
+
+---
+
+## Prerequisites
+
+**ACP Version Requirement: >= 4.0**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)